experimenter/experimenter/nimbus-ui/src/hooks/useCommonForm/index.test.tsx

```diff
@@ -149,7 +149,7 @@ describe("hooks/useCommonForm", () => {
     it("FormAudience", () => {
       render(<AudienceSubject />);
       const filteredAudienceFieldNames = audienceFieldNames.filter(
-        (e) => e !== "languages",
+        (e) => e !== "languages" && e !== "isFirstRun",
```

Comment:
same question- why this change?

Reply:
This change is not to check `isFirstRun` for the desktop which is the same case with `languages` so i added it here